dll/shellext/zipfldr/CZipFolder.hpp

```diff
@@ -351,9 +355,34 @@ class CZipFolder :
         switch (uMsg)
         {
         case DFM_MERGECONTEXTMENU:
-            DPRINT1("FIXME: Add menu items for DFM_MERGECONTEXTMENU\n");
+        {
+            QCMINFO *pqcminfo = (QCMINFO *)lParam;
+            CComPtr<IContextMenu> spContextMenu;
```

Comment:
`CComQIIDPtr`

Reply:
Thanks to GCC we can't use `CComQIPtr `

Reply:
Then you should test @yagoulas' CComQIIDPtr : https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=CComQIIDPtr

Reply:
@HBelusca please read the comments you are replying to.